website/docs/cloud-docs/cost-estimation/aws.mdx

```diff
@@ -12,19 +12,146 @@ Terraform Cloud can estimate monthly costs for many AWS Terraform resources.
 
 ## Handled Resources
```

Comment:
Changing this wording to "Supported Resources" because I believe that is normally how we describe things like this. Please let me know if "Handled" is more correct.

```suggestion
## Supported Resources
```